.github/workflows/qc_checks.yaml

```diff
@@ -114,7 +117,7 @@ jobs:
       - name: Enviroment Setup
         uses: ./.github/actions/setup
         with:
-          apt-dependency: gettext
+          apt-dependency: gettext poppler-utils
```

Comment:
should we mabybe add this requirement to the docs?

Reply:
Great point - I'll add this to https://github.com/inventree/inventree-docs/pull/295